code/modules/mob/inventory.dm

```diff
@@ -185,6 +185,22 @@
 		return hand_index || TRUE
 	return FALSE
 
+/mob/proc/force_put_in_hand(obj/item/I, hand_index)
+	if(hand_index == null)
+		return FALSE
+	if(get_item_for_held_index(hand_index) != null)
+		dropItemToGround(get_item_for_held_index(hand_index), force = TRUE)
+	I.forceMove(src)
+	held_items[hand_index] = I
+	I.layer = ABOVE_HUD_LAYER
+	I.plane = ABOVE_HUD_PLANE
+	I.equipped(src, slot_hands)
+	if(I.pulledby)
+		I.pulledby.stop_pulling()
+	update_inv_hands()
+	I.pixel_x = initial(I.pixel_x)
+	I.pixel_y = initial(I.pixel_y)

```

Comment:
why are you touching pixel vars and layering here?

Reply:
For the same reasons as put_in_hand()

Reply:
why are you not using put_in_hand then?

Reply:
It fails if used on a prone mob (like when laying down for surgery, a.k.a the whole reason for this bug) so oranges instructed me to make an alternate "forced" version without that check.

Reply:
https://github.com/tgstation/tgstation/blob/8893d3f100319b8927574628b777c85436c08a8f/code/modules/mob/inventory.dm#L173

You want to add an arg for that like `forced = FALSE`, replace 174 with `if(forced || can_put_in_hand(I, hand_index))` and that should do the exact same as _this_ . 0 need to copypasta a proc.